### PR TITLE
Adds abandoned status to proposal lifecycle

### LINKED ISSUE #770: Add "abandoned" status to proposal lifecycle

### PR DESCRIPTION
**Pull request by s-ben**
Adds step in the proposal lifecycle to explain how abandoned proposals work ( Closes #770 ).

>> 4. If a proposer owner decides to abandon their proposal (for instance, if comments from the community indicate it is not likely to pass), they can ask the admins to mark their proposal as ‘Abandoned’. Abandoned proposals appear in the ‘Abandoned’ tab on Politeia. They are publicly viewable, but cannot be edited, commented on, or be authorized to vote once abandoned. Admins can mark a proposal as abandoned at any time (for instance, if it is deemed spam) before the proposal owner authorizes voting. Once the proposal owner authorizes voting, it cannot be abandoned.

### CODE REVIEW

**Comment by RichardRed0x** [comment]
> If a proposer owner decides to abandon their proposal (for instance, if comments from the community indicate it is not likely to pass), they can ask the admins to mark their proposal as ‘Abandoned’.

This makes it sound like Abandoned status is something proposal owners ask for, but that's not really the case. It has mostly(entirely?) been used with proposals that were abandoned by their owners (i.e. they did not comment or stopped commenting, they did not authorize the start of voting, admin sets to Abandoned so we can get it out of the way).

>  Admins can mark a proposal as abandoned at any time (for instance, if it is deemed spam)
If it was deemed spam it would be censored.

> Once the proposal owner authorizes voting, it cannot be abandoned.
You can flip the switch back and forth, if you authorize then revoke that and disappear, it would still be made abandoned.

**Comment by s-ben** [comment]
How about this:

> 4. If a proposal becomes clearly inactive (proposal owner stops commenting on the proposal, does not authorize the start of voting), an admin may mark the proposal as ‘Abandoned’. Abandoned proposals appear in the ‘Abandoned’ tab on Politeia. They are publicly viewable, but cannot be edited, commented on, or be authorized to vote. An admin can mark a proposal as abandoned at any time before the proposal owner authorizes voting. Once the proposal owner authorizes voting on a proposal, it cannot be abandoned unless the proposal owner revokes voting on the proposal.

**Comment by jholdstock** [comment]
Requesting a review from @lukebp.

@s-ben as a side note, you had to increment numbers 5, 6, 7 and 8 in the list, so this PR affects lines which don't actually have any changes to the copy. That can be avoided by using `1.` on every line - https://www.markdownguide.org/basic-syntax/#lists

File: docs/governance/politeia/politeia.md [hunk 1]
```diff
@@ -35,11 +35,12 @@ Below is the basic process for submitting and evaluating proposals:
...
-7. When a proposal with a budget and deliverables is approved, work can begin. The proposal owner can submit claims against the budget as deliverables are completed.
-8. Payment claims will be handled manually by Decred Holdings Group (DHG) until such times as this process can be handled by the [Decred Decentralized Autonomous Entity (DAE)](#decentralized-autonomous-entity-dae).
+4. If a proposer owner decides to abandon their proposal (for instance, if comments from the community indicate it is not likely to pass), they can ask the admins to mark their proposal as ‘Abandoned’. Abandoned proposals appear in the ‘Abandoned’ tab on Politeia. They are publicly viewable, but cannot be edited, commented on, or be authorized to vote once abandoned. Admins can mark a proposal as abandoned at any time (for instance, if it is deemed spam) before the proposal owner authorizes voting. Once the proposal owner authorizes voting, it cannot be abandoned.
```
Comment (lukebp) [inline docs/governance/politeia/politeia.md]: > 4. If a proposer owner decides to abandon their proposal

Is "proposer owner" intended or is that suppose to be "proposal owner"?

> Admins can mark a proposal as abandoned at any time (for instance, if it is deemed spam) before the proposal owner authorizes voting.

Admins aren't suppose to mark a proposal as abandoned if it is spam.  The only time an admin should mark a proposal as abandoned is if the proposal has been inactive for a few weeks.  Spam shouldn't get approved in the first place.  If a proposal author gets their proposal approved, but then edits it to be spam, then the proposal will be censored.  The ability to censor public proposals hasn't been implemented yet.  There is a issue open for it though.

**Comment by s-ben** [comment]
@lukebp , good catches. Also good to know about the open issue for censoring proposals. I've subscribed to that Issue and will document if/when it's implemented. In my rewording (below), I tried to stick to what is only implemented now, and not get into any details still in flux or not specific enough (e.g. I don't think we should say "a couple weeks" in the docs, lest that change, or people get angry their proposal is abandoned in days or months instead). 

> If a proposal becomes clearly inactive (proposal owner stops commenting on the proposal, does not authorize the start of voting), an admin may mark the proposal as ‘Abandoned’. Abandoned proposals appear in the ‘Abandoned’ tab on Politeia. They are publicly viewable, but cannot be edited, commented on, or be authorized to vote. An admin can mark a proposal as abandoned at any time before the proposal owner authorizes voting. Once the proposal owner authorizes voting on a proposal, it cannot be abandoned unless the proposal owner revokes voting on the proposal.

@jholdstock am switching to `.1` for all future markdown lists. In addition to making it easier to add items, it will keep the +/- changed lines metric more useful; it's already pretty flawed, but still gets used occasionally.

**Comment by lukebp** [comment]
LGTM

**Pull request merged**